### DISCUSSION TITLE
Fundamentos do ASP.NET 6 - ViewModel

### DISCUSSION THREAD

**Discussion by allex-ribeiroh** [balta-io/2811#32] [Q&A] [answered]
Balta, boa noite.

No video sobre ViewModel, você comenta que a aplicação esta bem estruturada, mas porem não é uma modelo para aplicações grandes, sendo assim, qual o modelo/organização é indicado para aplicações grandes, consegue indicar um modelo por favor, quero ter uma ideia de como organizar / estruturar uma API para um cenário de uma aplicação grande.

**Comment by andrebaltieri** [comment, marked as answer]
@allex-ribeiroh bom dia!!!
Na verdade não existe um modelo... cada caso é um caso e neste cenário, o uso de ViewModels e MVC é a estrutura mais concisa para a aplicação.

Arquiteturas evoluem junto com o software... nos cursos 1975 e 7182 você verá sobre CQRS e conceitos mais avançados.

**Reply by allex-ribeiroh** [reply]
Grato Balta
